test/config.js

```diff
@@ -2,7 +2,6 @@ const forEachObjIndexed = require('ramda/src/forEachObjIndexed')
 
 // we set config for amio+sdk_tests@amio.io in prod
 const config = {
-  CONNECTOR_API_BASE_URL: 'https://api.amio.io',
```

Comment:
I would not mind to have it here rather then hardcoded somewhere in code..

Reply:
It is just a config for tests. I will keep it where it is.